Source/core/JSON.h

```diff
@@ -1691,8 +1691,15 @@ namespace Core {
                             
                             if (result < maxLength) {
 
+                                char convertedValue = IsEscapeSequenceValue(*source);
+                                if (convertedValue != 0) {
+                                    stream[result++] = '\\';
```

Comment:
This might fail if you run the Small parser test (only 1 character at a time in the buffer). See this testcase:
https://github.com/rdkcentral/Thunder/blob/master/Tests/unit/core/test_json.cpp#L137
this is testing with a buffer size of1. COuld we test this feature with this tester if it still works ?

Reply:
yes, this test passed. I ran all unit test cases with this changes and all are passed

Reply:
all test cases ran with the changes and passes